pyaedt/edb_core/stackup.py

```diff
@@ -526,66 +528,7 @@ def remove_layer(self, name):
         return self._pedb._active_layout.SetLayerCollection(new_layer_collection)
 
     @pyaedt_function_handler
-    def import_stackup(self, fpath):
-        """Import stackup defnition from csv file.
-
-        Parameters
-        ----------
-        fpath : str
-            File path to csv or json file.
-        """
-        if not pd:
-            self._pedb.logger.error("Pandas is needed. Please, install it first.")
-            return False
-        if os.path.splitext(fpath)[1] == ".json":
-            return self._import_layer_stackup(fpath)
-        if is_ironpython:
-            self._pedb.logger.error("Method working on CPython only.")
-            return False
-        df = pd.read_csv(fpath, index_col=0)
-        prev_layer = None
-        for row, val in df[::-1].iterrows():
-            if not self.stackup_layers:
-                self.add_layer(
-                    row,
-                    None,
-                    "add_on_top",
-                    val.Type,
-                    val.Material,
-                    val.Dielectric_Fill if not pd.isnull(val.Dielectric_Fill) else "",
-                    val.Thickness,
-                )
-            else:
-                if row in self.stackup_layers.keys():
-                    lyr = self.stackup_layers[row]
-                    lyr.type = val.Type
-                    lyr.material = val.Material
-                    lyr.dielectric_fill = val.Dielectric_Fill if not pd.isnull(val.Dielectric_Fill) else ""
-                    lyr.thickness = val.Thickness
-                    if prev_layer:
-                        self._set_layout_stackup(lyr._edb_layer, "change_position", prev_layer)
-                else:
-                    if prev_layer and prev_layer in self.stackup_layers:
-                        layer_name = prev_layer
-                    else:
-                        layer_name = list(self.stackup_layers.keys())[-1] if self.stackup_layers else None
-                    self.add_layer(
-                        row,
-                        layer_name,
-                        "insert_above",
-                        val.Type,
-                        val.Material,
-                        val.Dielectric_Fill if not pd.isnull(val.Dielectric_Fill) else "",
-                        val.Thickness,
-                    )
-                prev_layer = row
-        for name in self.stackup_layers:
-            if name not in df.index:
-                self.remove_layer(name)
-        return True
-
-    @pyaedt_function_handler
-    def export_stackup(self, fpath, file_format="csv", include_material_with_layer=False):
+    def export_stackup(self, fpath, file_format="xml", include_material_with_layer=False):
         """Export stackup definition to a CSV or JSON file.
 
         Parameters
```

Comment:
In line 537, use capital letters for file extension types (CVS or JSON, not csv or json). There are other occurrences that should also be changed. The rule is when you are specifying a specific file, surround in double back ticks to tag (such as ``test.csv``.  When the extension is used without a file name, you capitalize the extension.